features/edit_membership_application.feature

```diff
@@ -0,0 +1,31 @@
+Feature: As an applicant
+  In order to be able to edit my application
+  I want to be allowed to do that
+
+  PT Feature: https://www.pivotaltracker.com/story/show/134078325
```

Comment:
Could you make this on line four and just PT: link ? 
The second feature gets coloured in some ide(?)'s and that could be confusing.